playbooks/aws_role/create.yml

```diff
@@ -6,7 +6,7 @@
         name: "{{ role_name }}"
         state: present
         description: "{{ role_description }}"
-        assume_role_policy_document: "{{ lookup('file','/tmp/policy.json') }}"
+        assume_role_policy_document: "{{ lookup('file','/tmp/resources/policy.json') }}"

```

Comment:
Same as before. I do not see the reason to change this in the current repo just so the circleci step will be easier.

Reply:
Fair enough! I can redo that.